src/main/java/com/puppycrawl/tools/checkstyle/api/FileText.java

```diff
@@ -52,7 +51,7 @@
  *
  * @author Martin von Gagern
  */
-public final class FileText extends AbstractList<String> {
+public final class FileText {
```

Comment:
Please, correct javadoc. It should not mention [List interface](https://github.com/checkstyle/checkstyle/pull/4525/files/aa2d11976ebfa0f9c95ad4de1f879d9a49d41ebb#diff-e0e48a646884b6e6bb965ac3740984baR46).